lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -773,11 +772,15 @@ def create_autoscaling_group(connection, module):
     health_check_type = module.params.get('health_check_type')
     default_cooldown = module.params.get('default_cooldown')
     wait_for_instances = module.params.get('wait_for_instances')
-    as_groups = connection.describe_auto_scaling_groups(AutoScalingGroupNames=[group_name])
     wait_timeout = module.params.get('wait_timeout')
     termination_policies = module.params.get('termination_policies')
     notification_topic = module.params.get('notification_topic')
     notification_types = module.params.get('notification_types')
+    try:
+        as_groups = connection.describe_auto_scaling_groups(AutoScalingGroupNames=[group_name])
```

Comment:
Should this be a call to a function that does handle retries (all the other API calls in this function seem to at least be capable of that)